test/OpenTelemetry.Tests/Impl/Trace/TracerTest.cs

```diff
@@ -410,8 +410,8 @@ public void Tracer_StartSpanFromActivity_ValidArguments_ReturnsSpanWithActivityA
 
             Assert.NotNull(span);
 
-            Assert.NotNull(span.Attributes.FirstOrDefault(k => k.Key == "key1"));
-            Assert.NotNull(span.Attributes.FirstOrDefault(k => k.Key == "key2"));
+            Assert.Equal("value1", span.Attributes.FirstOrDefault(k => k.Key == "key1").Value);
+            Assert.Equal("value2", span.Attributes.FirstOrDefault(k => k.Key == "key2").Value);
```

Comment:
I'd suggest In this case to use `First`. To indicate you expect an item, since `FirstOrDefault` would return `null` and when null-reference types is introduced this will give an error.

```suggestion
            Assert.Equal("value1", span.Attributes.First(k => k.Key == "key1").Value);
            Assert.Equal("value2", span.Attributes.First(k => k.Key == "key2").Value);
```

Reply:
Thanks for the help

Reply:
Thank you!